lcm/service/lcm/container_helper_extensions.go

```diff
@@ -31,6 +31,8 @@ func extendLearnerContainer(learner *v1core.Container, req *service.JobDeploymen
 		learnerImage = "caffe2ai/caffe2:" + req.Version
 	case pytorchFrameworkName:
 		learnerImage = "pytorch/pytorch:" + req.Version
+	case h2o3FrameworkName:
+		learnerImage = "nkpng2k/h2o3:" + req.Version
```

Comment:
Can this come froma more generic account than nkpng2k?

Reply:
This is just a temporary image for testing. I have asked nick for h2o-3's stable image and we will replace this with the new image.